wear/src/main/java/com/eveningoutpost/dexdrip/Services/CustomComplicationProviderService.java

```diff
@@ -129,34 +132,56 @@ public void onComplicationUpdate(
                 is_stale = true;
             }
         }
+
+        //Loop status by @gregorybel
+        String externalStatusString = PersistentStore.getString("remote-status-string");
```

Comment:
Can you move this `PersistentStore` lookup in to the `case ComplicationData.TYPE_LONG_TEXT:` section to avoid the overhead of the lookup when the user is not using the `TYPE_LONG_TEXT` ?

Reply:
I use it too for SHORT_TEXT so it believe no.